app/javascript/retrospring/features/answerbox/subscribe.ts

```diff
@@ -0,0 +1,43 @@
+import Rails from '@rails/ujs';
+
+import I18n from '../../../legacy/i18n';
+import { showNotification, showErrorNotification } from 'utilities/notifications';
+
+export function answerboxSubscribeHandler(event: Event): void {
+  const button = event.target as HTMLButtonElement;
+  const id = button.dataset.aId;
+  let torpedo = 0;
+  let targetUrl;
+  event.preventDefault();
+
+  if (button.dataset.torpedo === 'yes') {
+    torpedo = 1;
+  }
+
+  if (torpedo) {
+    targetUrl = '/ajax/subscribe';
+  } else {
+    targetUrl = '/ajax/unsubscribe';
+  }
+
+  Rails.ajax({
+    url: targetUrl,
+    type: 'POST',
+    data: new URLSearchParams({
+      answer: id
+    }).toString(),
+    success: (data) => {
+      if (data.success) {
+        button.dataset.torpedo = ["yes", "no"][torpedo];
+        button.children[0].nextSibling.textContent = torpedo ? I18n.translate('views.actions.unsubscribe') : I18n.translate('views.actions.subscribe');
```

Comment:
Also missed this one
```suggestion
        button.children[0].nextSibling.textContent = ' ' + (torpedo ? I18n.translate('views.actions.unsubscribe') : I18n.translate('views.actions.subscribe'));
```